src/CommunityToolkit.Maui.UnitTests/Converters/IntToBoolConverter_Tests.cs

```diff
@@ -25,16 +27,17 @@ public void IndexToArrayConverterBack(bool value, int expectedResult)
 	{
 		var intToBoolConverter = new IntToBoolConverter();
 
-		var result = (int)intToBoolConverter.ConvertBack(value, typeof(bool), null, CultureInfo.CurrentCulture);
+		var result = intToBoolConverter.ConvertBack(value, typeof(bool), null, CultureInfo.CurrentCulture);
+		var typedResult = intToBoolConverter.ConvertBackTo(value);
 
 		Assert.Equal(result, expectedResult);
+		Assert.Equal(typedResult, expectedResult);
```

Comment:
Let's swap these `Assert.Equal` parameters around.

It's kinda trivial, but when the `Assert` fails, it'll print "...Expected: [first parameter] Actual: [second parameter]" which can be confusing if the parameters are reversed.

```suggestion
		Assert.Equal(expectedResult, result);
		Assert.Equal(expectedResult, typedResult);
```